CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing Guidelines
+
+Cycloid Team is glad to see you contributing to this project ! In this document, we will you provide some guidelines in order to help get your contribution accepted.
+
+## Reporting an issue
+
+When you find a bug in Terracognita, it should be reported using [GitHub issues](https://github.com/cycloidio/terracognita/issues). Please define key information like your Operating System (OS), Terracognita origin (docker or from source) and finally the version you are using.
+
+## Submit a contribution
+
+### Setup your git repository
+
+If you want to contribute to an existing issue, you can start by _fork_ this repository, then clone your fork on your machine.
+
+```shell
+$ git clone https://github.com/<your-username>/terracognita.git
+$ cd terracognita
+```
+
+In order to stay updated with the upstream, it's highly recommended to add cycloidio/terracognita as a remote upstream.
+
+```shell
+$ git remote add upstream https://github.com/cycloidio/terracognita.git
+```
+
+Do not forget to frequently update your fork with the upstream.
+
+```shell
+$ git fetch upstream --prune
+$ git rebase upstream/master
+```
+
+### Go dependencies
+
+Since Terracognita is a Go project, Go must be installed and configured on your machine (really ?). We currently support Go1.12 and Go.13 and go `modules` as dependency manager. You can simply pull all necessaries dependencies by running an initial 
```

Comment:
Actually, I believe one could simply build a docker image to test - can you confirm?

Reply:
Indeed, the contributor could build a Docker image to test Terracognita. However, we are writing a contributing guide, not a readme ?
If the user wants to contribute, he first has to pull dependencies and setup his environment.
Or I misunderstood what did you say. :wink: